bin/docker-entrypoint.sh

```diff
@@ -109,20 +109,26 @@ if [ ! "$(stat -c '%U' /var/log/odoo)" = "odoo" ]; then
 fi
 
 BASE_CMD=$(basename $1)
-if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ] ; then
-
-  BEFORE_MIGRATE_ENTRYPOINT_DIR=/before-migrate-entrypoint.d
-  if [ -d "$BEFORE_MIGRATE_ENTRYPOINT_DIR" ]; then
-    run-parts --verbose "$BEFORE_MIGRATE_ENTRYPOINT_DIR"
-  fi
+CMD_ARRAY=($*)
+ARGS=(${CMD_ARRAY[@]:1})
 
-  if [ -z "$MIGRATE" -o "$MIGRATE" = True ]; then
-    gosu odoo migrate
-  fi
+if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ] ; then
 
-  START_ENTRYPOINT_DIR=/start-entrypoint.d
-  if [ -d "$START_ENTRYPOINT_DIR" ]; then
-    run-parts --verbose "$START_ENTRYPOINT_DIR"
+  # Bypass entrypoints when `odoo shell` or `odoo --help` are used
```

Comment:
While bypassing migrate makes sense, I don't think bypassing entrypoints is a  good idea. Some entrypoints *have* to be run before odoo is executed in dev (disable email server for instance) and so they must be applied for shell as well.

Reply:
Ok, I'll limit to the migration and keep the entrypoints.